content/pages/changelog/index.md

```diff
@@ -13,6 +13,23 @@ For historical changelog entries, please see our
 [2019](page://changelog/2019) and
 [2018 release notes](page://changelog/2018).
 
+## 2020-06-11 / New enterprise events for tasks
+
+Starting today, the [enterprise event stream](g://events/for-enterprise/) will
+begin producing new task and task assignment events and some existing task
```

Comment:
I'd add a comma after events to signify that this is a different `and` than the previous one.